spark/src/main/scala/ai/chronon/spark/Analyzer.scala

```diff
@@ -184,11 +184,12 @@ class Analyzer(tableUtils: TableUtils,
     AggregationMetadata(columnName, columnType, "No operation", "Unbounded", columnName)
   }
 
-  def analyzeGroupBy(groupByConf: api.GroupBy,
-                     prefix: String = "",
-                     includeOutputTableName: Boolean = false,
-                     enableHitter: Boolean = false,
-                     skipTimestampCheck: Boolean = false): (Array[AggregationMetadata], Map[String, DataType]) = {
+  def analyzeGroupBy(
+      groupByConf: api.GroupBy,
+      prefix: String = "",
+      includeOutputTableName: Boolean = false,
+      enableHitter: Boolean = false,
+      skipTimestampCheck: Boolean = skipTimestampCheck): (Array[AggregationMetadata], Map[String, DataType]) = {
```

Comment:
@yuli-han - shouldn't this be Boolean = false?

Reply:
Hi @pkundurthy I already set the skipTimestampCheck =  true when initialize the analyzer https://sourcegraph.a.musta.ch/github.com/airbnb/chronon/-/blob/spark/src/main/scala/ai/chronon/spark/MetadataExporter.scala?L53 If we set default value to be false we have to pass the skipTimestampCheck again when calling the analyzeJoin function. It should be equal to the skipTimestampCheck default value when we initialize the analyzer right?